spec/system/homepage/user_visits_homepage_spec.rb

```diff
@@ -29,8 +30,28 @@
       expect(page).to have_text("DESIGN YOUR EXPERIENCE")
     end
 
+    context "when rendering broadcasts" do
+      let!(:broadcast) { create(:announcement_broadcast) }
+
+      it "renders the broadcast if active", js: true do
+        get "/async_info/base_data" # Explicitly ensure broadcast data is loaded before doing any checks
```

Comment:
I don't love that I have to do this, but it seems necessary since, afaik, the request to `async_info/base_data` doesn't happen by default. If someone has better ideas, open to hearing them!